demo/src/App.jsx

```diff
@@ -153,8 +154,54 @@ class App extends Component {
               </Flyout.Content>
             </Flyout.Wrapper>
             <Button type="submit" size="md" theme="beta">
-            Button test
-          </Button>
+              Button test
+            </Button>
+            <h3>Button Dropdowns</h3>
+            <ButtonDropdown.Wrapper togglePosition="left">
```

Comment:
API for button dropdown looks good.  This is pretty much what we want to present to consumers.